OWNERS_ALIASES

```diff
@@ -29,9 +29,10 @@ aliases:
     - erikerlandson
     - liyinan926
   sig-cli-leads:
+    - soltysh
```

Comment:
I don't need to be called out twice, do I ;)

Reply:
This is because you're listed as both a chair and a tech lead. The aliases are autogenerated (it's just a dumb go text/template, so it doesn't de-dupe).